openshift_metrics/metrics_processor.py

```diff
@@ -65,8 +71,22 @@ def _extract_gpu_info(metric_name: str, metric: Dict) -> GPUInfo:
             gpu_resource = metric["metric"].get("resource")
             node_model = metric["metric"].get("label_nvidia_com_gpu_machine")
 
+            # Sometimes GPU labels from the nodes can be missing, in that case
+            # we get the gpu_type from the gpu-node file
+            if gpu_type == GPU_UNKNOWN_TYPE:
+                node_name = metric["metric"].get("node")
+                gpu_type = self.gpu_mapping.get(node_name, GPU_UNKNOWN_TYPE)
+
         return GPUInfo(gpu_type, gpu_resource, node_model)
 
+    @staticmethod
+    def _load_gpu_mapping(file_path: str) -> Dict[str, str]:
+        try:
+            with open(file_path, "r") as file:
+                return json.load(file)
+        except FileNotFoundError:
+            return {}
```

Comment:
I advise logging a warning here, since the filename might unintentionally have a typo and the operator would not have any indication as to what's wrong.